src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Network Analysis/NavigateRoute/NavigateRoute.xaml

```diff
@@ -0,0 +1,48 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.UWP.Samples.NavigateRoute.NavigateRoute"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esriUI="using:Esri.ArcGISRuntime.UI.Controls">
+    <Grid>
+        <esriUI:MapView x:Name="MyMapView" />
+        <Border Style="{StaticResource BorderStyle}">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition />
+                    <RowDefinition Height="100" />
+                </Grid.RowDefinitions>
+                <Grid>
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition />
+                        <ColumnDefinition />
+                    </Grid.ColumnDefinitions>
+                    <Button
+                        x:Name="StartNavigationButton"
+                        Grid.Column="0"
+                        Margin="5"
+                        HorizontalAlignment="Stretch"
+                        VerticalAlignment="Center"
+                        Click="StartNavigation"
+                        Content="Navigate Route" />
```

Comment:
I recommend making 'Route' lower case here to be consistent with the Runtime style.